Workbooks/WorkbooksMetadata.json

```diff
@@ -1415,5 +1415,18 @@
       "templateRelativePath": "AzureSentinelCost.json",
       "subtitle": "",
       "provider": "Azure Sentinel Community"
+    },   
+    {
+      "workbookKey": "AzurePurview",
+      "logoFileName": "AzurePurview.svg",
+      "description": "This workbook provides insights into the sensitivity of data that has been scanned by Azure Purview, including classifications and labels found across the data estate.",
+      "dataTypesDependencies": ["PurviewDataSensitivityLogs"],
+      "dataConnectorsDependencies": ["AzurePurview"],
+      "previewImagesFileNames": ["AzurePurviewWhite1.png", "AzurePurviewBlack1.png", "AzurePurviewWhite2.png", "AzurePurviewBlack2.png","AzurePurviewWhite3.png", "AzurePurviewBlack3.png"],
```

Comment:
Please maintain consistency in the file extensions, while creating and mentioning it in Metadata file. This is one of the reason for validation failure.

e.g. File is added with name as  **AzurePurviewWhite1.PNG** under preview folder and is being referenced as **AzurePurviewWhite1.png** in metadata file. 

Please fix it for all the images.

Thanks!!!

Reply:
Fixed the file extensions for the images in the preview folder.